### DISCUSSION TITLE
ERA5-Land Reanalysis request

### DISCUSSION THREAD

**Discussion by TomAugspurger** [microsoft/PlanetaryComputer#21] [Data request]
The Planetary Computer already hosts several variables from ERA5. This discussion is for ERA5 land.

Source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-land?tab=overview
Temporal frequency: Hourly
Temporal extent: 1950 - present
Spatial extent: global.
Format: GRIB2 or NetCDF. Likely converted from NetCDF to Zarr following what we do for era5.
Variables:

- total precipitation
- tmin
- tmax
- shortwave rad
- wind speed
- humidity
- vapor pressure
- volumetric water content
- volumetric soil water
- snow water equivalent
- surface runoff
- sub-surface runoff
- snow melt
- snow fall
- snow cover
- all volumetric soil water layers (1-4)
- total evapotranpiration

---

Dataset: [ERA5 Reanalyse](https://www.ecmwf.int/en/forecasts/dataset/ecmwf-reanalysis-v5)

(copied from https://github.com/microsoft/PlanetaryComputer/discussions/5#discussioncomment-1913340)

**Comment by TomAugspurger** [comment]
cc @WalternativE. Is there a specific subset of variables that you'd need for your work? Are the ones from planet-os / S3 documented at https://github.com/planet-os/notebooks/blob/master/aws/era5-pds.md sufficient?

**Reply by WalternativE** [reply]
The variables from the PlanetOS documentation would be sufficient. There are a bunch of additional ones that would be useful for more intricate hydrometric models I assume (I'm not an expert in this field but especially runoff and volumetric soil water in the different layers appear very useful). Personally, I'd be interested into all what's already present as there are plenty of variables that can be used to correlate with time series of renewable energy production. Thanks for opening this issue and additional thanks for looking into adding this to the catologue 🙏

**Comment by rbavery** [comment]
We have an ongoing project to improve the user interface to the [Aquacrop](https://github.com/thomasdkelly/aquacrop) crop model so that users don't need to manually fetch the numerous climate and other model inputs. 

[ERA-5 Land](https://www.ecmwf.int/en/era5-land) (different from ERA-5) would be a very useful addition to the PC Computer which would allow us to run the model on a finer resolution grid than the CMIP 6 product. The CMIP6 product is also limited in that it only uses historical, station-based data until 2014, at which point it switches to model projections.

We would like PC access to the following daily global variables: precip, tmin, tmax, shortwave rad, wind speed, humidity or vapor pressure, and volumetric water content.

**Comment by azahradka** [comment]
+1 for ERA5-Land, it would be an extremely high-value addition for our work in the applied earth sciences. In addition to the above variables, we frequently use volumetric soil water, snow water equivalent, surface runoff, snow melt, and snow fall.

**Comment by rsignell-usgs** [comment]
We are also very interested in ERA5-Land here at USGS.   
 Of course we would like the ability to be able to access all variables (!) but if we had to rank them they would be:

* All volumetric soil water layers (1 through 4)
* Total evapotranspiration, total precipitation
* Sub-surface runoff, surface runoff, snow depth water equivalent
* Everything!

@sfoks

**Reply by sfoks** [reply]
If we can add 'snow cover' to the third bullet, that would be appreciated! Thank you!

**Reply by rmwebb-usgs** [reply]
Plus snowmelt. Initiation of snowmelt occurs when the pack is isothermal so any variable indicating negative heat deficit for the pack would be great. Also any human water use such as detention is reservoirs, and irrigation from diversions and aquifers. Is ERA-5-Land, at a resolution of around 9-km, able to reproduce intense convective storms, that need a resolution of around 4-km (recent work at NCAR and elsewhere)?

**Comment by smartfarming-udf** [comment]
+1 for ERA5-Land. Would need SSRD (surface solar radiation downwards).

**Comment by jacobbieker** [comment]
Also interested in ERA5-Land, especially solar and wind variables for renewable energy work.
